### PR TITLE
change readme

### PR DESCRIPTION
**Pull request by YJ2CS**
hey, guy! You should change your README. In this way, people can install your npm package normally after reading the README.
In addition, your npm "Homepage" and "Repository" should also be modified.

It should be modified to `https://github.com/GerHobbelt/markdown-it-wikilinks#readme`

image

image

Also, these two buttons should also be replaced or deleted. They point to the address of the original author, not to your npm package.

[Coverage Status](https://coveralls.io/github/jsepia/markdown-it-wikilinks?branch=master)

[Build Status](https://travis-ci.org/jsepia/markdown-it-wikilinks) 

```txt
[![Coverage Status](https://coveralls.io/repos/github/jsepia/markdown-it-wikilinks/badge.svg?branch=master)](https://coveralls.io/github/jsepia/markdown-it-wikilinks?branch=master)

[![Build Status](https://travis-ci.org/jsepia/markdown-it-wikilinks.svg?branch=master)](https://travis-ci.org/jsepia/markdown-it-wikilinks) 
```

### CODE REVIEW

**Comment by YJ2CS** [comment]
Hello developer, please forgive me for using *pull requests* as *issue*, because I didn't find a suitable contact method.

I am sending you a request to add two **new features,**

## Features 1.

This is a grammar that extends wikilinks recently.

Similar to the following form

`<img_path>` default is `/images`

`![[image.png|HelloWorld]]` -> `/img.png" alt="HelloWorld" />`

The simpler form is,
`![[image.png]]` -> `/img.png" />`

There is a `.png` suffix to specify the suffix name of this picture, and,

If you add a `!` before `[[wiki]]`, it becomes `![[wiki]]`, then it is considered that this will be parsed as an `HTML ` tag.

This syntax is very similar to traditional wikilink. It should be noted that it is necessary to determine whether there is a `!`

## Features 2.

In some specific use environments, such as the blog I write now, the following form is required.

`[[wiki link|here]]` -> `<a href="./Wiki-Links.html">here</a>`

Simply put, the space character should be replaced with `-` instead of the current `_`. I think maybe a new option can be added to allow users to customize this replacement character
